Mango API/api-test/fileStore.spec.js

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-const {createClient, login} = require('@infinite-automation/mango-module-tools/test-helper/testHelper');
```

Comment:
@benji811124 this won't work unless `@radixiot` package added to package.json and installed. I think we don't need to backport this tests and module-tools upgrade as long as manual QA passed for this version.

API tests running daily on Jenkins and going to fail after merging this PR.

Reply:
Am I reading this incorrectly, or does the fix for the filestore path traversal bug remove the ability to write files within subdirectories of the file store?

![image](https://github.com/MangoAutomation/ma-modules-public/assets/31995798/32f1ab72-3ce7-4ff6-aeb9-ff80e0d87571)

Reply:
Thanks for catching that. I'll fix it.

Reply:
@benji811124 are you running the tests? I wouldn't upgrade to the `@radixiot` version as it may break things, and you can't use the `@infinite-automation` version because it doesn't have the ability to POST FormData.
I agree with @MertCingoz that we don't need to backport the tests.

Reply:
I just ran a functional test last night. I'll remove the test from mango 4.5

Reply:
I've reverted the test commit